### PR TITLE
Regenerate module

### PR DESCRIPTION
**Pull request by lucky091588**

### CODE REVIEW

**Comment by lucky091588** [comment]
@twomice Please check PR. Thanks.

File: paaosumfields.civix.php [hunk 1]
```diff
@@ -6,10 +6,10 @@
  * The ExtensionUtil class provides small stubs for accessing resources of this
  * extension.
  */
-class CRM_Paaosumfields_ExtensionUtil {
+class CRM_Poaasumfields_ExtensionUtil {
```
Comment (stevekessler) [inline paaosumfields.civix.php]: @lucky091588 , I am not sure I understand why you changed the spelling of the class.  It should be PAAO.

Thanks,
Steve

Comment (lucky091588) [reply paaosumfields.civix.php]: Weird. I only did `civix generate:module`. I will double-check this. Thanks.

Comment (lucky091588) [reply paaosumfields.civix.php]: It seems that when you do `civix generate:module` the civix will base the changes in `info.xml` `<namespace>` which has the wrong spelling. Fixing this on the latest commit.

**Comment by twomice** [comment]
Thanks @lucky091588

**Pull request merged**